tests/screenshot/apps/components/DayPicker.js

```diff
@@ -26,6 +26,25 @@ const DayPickerTests = [
 		component: <DayPicker disabled selected={1} />
 	},
 
+	// *************************************************************
+	// locale = 'es-ES', [QWTC-637]
```

Comment:
@dongsuwon, Is the TC ticket number correctly related to these screenshot tests?

Reply:
@MikyungKim 
Yes, It is to check parts that cannot be checked in ui-test.
e.g) color of check/uncheck state, color of disable state